examples/kafka/Dockerfile.example.kafka

```diff
@@ -0,0 +1,14 @@
+FROM pai.build.base:hadoop2.7.2-cuda8.0-cudnn6-devel-ubuntu16.04
+
+#install python3 and vim
+RUN apt-get update && apt-get install python3.4 && apt-get install vim
```

Comment:
Is `vim` needed?

Reply:
I think user need a tool to edit their start shell file and logic code to use kafka, so I install vim here.